MFVNeutralino/test/TrackMover/background_leptonpresel_20161.root.haddlog.2

```diff
@@ -0,0 +1,7 @@
+Warning in <TInterpreter::ReadRootmapFile>: class  edm::RefProd<vector<pat::Jet> > found in libDataFormatsPatCandidates.so  is already in libJMTuckerMFVNeutralinoFormats.so 
+hadd Target file: background_leptonpresel_20161.root
+hadd compression setting for all ouput: 1
+hadd Source file 1: wjetstolnu_leptonpresel_20161.root
+Error in <TFile::TFile>: file wjetstolnu_leptonpresel_20161.root does not exist
+Error in <TFileMerger::AddFile>: cannot open file wjetstolnu_leptonpresel_20161.root
+hadd exiting due to error in wjetstolnu_leptonpresel_20161.root
```

Comment:
Let's delete this from the repo, we don't need to track it.

Reply:
Done.

Reply:
I still see it, did this not get pushed?